.github/workflows/build.yaml

```diff
@@ -378,10 +379,63 @@ jobs:
         name: Widelands ${{ matrix.config }} ${{ matrix.compiler }} MacOS 11 AppImage
         path: ${{ github.workspace }}/Widelands-${{ github.sha }}-${{ matrix.config }}-${{ matrix.compiler }}.dmg
 
+  appimage:
+# TODO enable before merging
+#    if: ${{ github.repository == 'widelands/widelands' && github.ref == 'refs/heads/master' }}
+    needs: [clang_tidy, documentation, codecheck, misc_sanity_checks, lua_style]
+    strategy:
+      matrix:
+        config:
+          - Release
+          - Debug
+        compiler:
+          - clang-10
+    # Must run on the oldest still-supported Ubuntu LTS release
+    runs-on: ubuntu-18.04
+    env:
+      VERSION: "${{ github.sha }}-${{ matrix.config }}-${{ matrix.compiler }}"
+    steps:
+    - name: Installing dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install git cmake ${{ matrix.compiler }} gettext libasio-dev libglew-dev libpng-dev libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev python3 zlib1g-dev -y      
```

Comment:
@matthiakl Assuming this will get merged after #5415, the dependencies can be installed via the script as shown here on L172 https://github.com/widelands/widelands/pull/5415/files#diff-d0777657fa3fd81d23aaf7273e58aee453b04e67882517900c56daeef9b3e4c1R172

Reply:
Thanks for the info. I updated the build file accordingly.